lib/reverse_markdown/converter_registry.rb

```diff
@@ -0,0 +1,39 @@
+module ReverseMarkdown
+  class ConverterRegistry
+    def initialize(config)
+      @config = config
+      @default_converter = create_default_converter
+      @default_converter.converters = self
+      @converters = {}
+    end
+
+    def register(tag_names, converter)
+      converter.converters = self
+
+      Array(tag_names).each do |tag_name|
+        @converters[tag_name.to_sym] = converter
+      end
+    end
+
+    def lookup(tag_name)
+      @converters[tag_name.to_sym] or @default_converter
+    end
+
+    private
+
+    def create_default_converter
+      case @config.unknown_tags.to_sym
```

Comment:
```suggestion
      case @config.unknown_tags&.to_sym
```

nil does not respond to to_sym. i would rather see the invalid config error then nil does not respond to to_sym